### ISSUE TITLE
I don't understand why you need algo files when js would be just fine

### ISSUE THREAD

**Issue by totty90** [blinktrade/algorithm-trading#1]

**Comment by pinhopro** [comment]
That was a bad decision. Hard to change now :/

**Comment by totty90** [comment]
I've noticed this and other issues. I would suggest to look into webpack + gulp to improve your code structure. 

Also another tip would be to use standard names for things. For example classes are named like this: `MyBot.js´ and not`my_bot.js`. Also the instance of the class is like this`var myBot = new MyBot()`. 

I also don't understand the methods and properties with a ending `_`. I use these coding practices:
- Private method/property: `__myMethod`;
- Protected: `_myMethod`;
- Public: `myMethod`;

**Comment by hems** [comment]
yeah all the examples are barely readable, looks like some compiled code ( ;

surely not helpful for beginners, things must and could be way more simple than they are..

Needless to say the idea of running algorithms on the user browser is fun and theoretically safe but definitely not ready for production, anyone wanting to earn money running algorithms should simply ignore this option and use the exchange API.

BTW the went to http://foxbit.exchange and clicked "API" on the bottom of the page and the link is broken..

good luck sorting this things out, it won't be easy.

peace

**Comment by cesardeazevedo** [comment]
Hi @hems, can you tell me what's going on about the "API" link? I didn't have any problems, could you create an issue on our [frontend repository](https://github.com/blinktrade/frontend)?

Regarding the algorithm trading, we're currently working on a client javascript sdk and a better interface to run algorithms that enables you running them on a local server...

**Comment by hems** [comment]
@cesardeazevedo when i click "API" on https://foxbit.exchange it just links me to a #, i didn't check the other websites. Probably not a blinktrade problem, my bad!
